src/main/g8/README.md

```diff
@@ -196,18 +207,19 @@ object Main {
 
 ## [scala-logging](https://github.com/lightbend/scala-logging) - universal logging ##
 
-`scala-logging` is a logging adapter specific to Scala that uses the [slf4j logging interface](https://www.slf4j.org/manual.html).  In turn, that interface is hooked up to [Log4J as the logging mechanism](http://logging.apache.org/log4j/2.x/manual/layouts.html) (the thing that actually logs), which has been sensibly preconfigured.
+`scala-logging` is a logging adapter specific to Scala that uses the [slf4j logging interface](https://www.slf4j.org/manual.html).  In turn, that interface is hooked up to [logback as the logging mechanism](http://logback.qos.ch/) (the thing that actually logs), which has been sensibly preconfigured.
 
 ###### Why do we use it? ######
 
 - `scala-logging` gives you a limited subset of the functionality (only the stuff you need).
 - `scala-logging` is coupled tightly to scala, and thus is very performant, making use of scala macros to intelligently not log when the logging level is not requested.  This allows us to leave trace & debug logs everywhere we need them without worrying about performance.
 - Abstraction over `slf4j` allows the end-user to decide how they want to handle logs.
-- `Log4J2` can be drop-in-replaced - all of the dependencies are run-time.  If you want to use logback (or whatever logging mechanism), simply bring along the dependency in the environment you wish to run in, and configure it.
+- `logback-classic` can be drop-in-replaced - all of the dependencies are run-time.
+- If you want to use `Log4J2` (or another logging backend that relies heavily upon reflection), be warned that the SubstrateVM's "out of the box" configuration won't be enough and [you'll need to do some configuration first](https://github.com/oracle/graal/issues/808#issuecomment-470769285). If you don't absolutely need log4j2, you could simply wait for SubstrateVM's [automatic detection of reflection](https://github.com/oracle/graal/blob/master/substratevm/REFLECTION.md#automatic-detection) to be expanded upon so you can keep your project a little more lean in the meantime.     
 
 ###### How do we use it? ######
 
-- Modify `src/main/resource/log4j2.xml` to your desire for local development. It currently has good defaults for moderate log usage in production (20 files rotating, 25 MB max each, INFO level)
+- Modify `src/main/resources/logback.xml` to your desire. It currently has a basic, yet colorful, logger that simply prints TRACE level and above to the console.
```

Comment:
Good callout that this logger config has too much pumpkin spice.

I'll branch later and add support that Log4j2 was providing in the previous project, [but in a better way](http://logback.qos.ch/apidocs/ch/qos/logback/core/rolling/SizeAndTimeBasedRollingPolicy.html).

Reply:
Issue created: https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/issues/2

Reply:
👏 👏 👏